website/content/docs/nia/enterprise/index.mdx

```diff
@@ -0,0 +1,14 @@
+---
+layout: docs
+page_title: Consul-Terraform-Sync Enterprise
+description: >-
+  Consul-Terraform-Sync Enterprise
+---
+
+# Consul-Terraform-Sync Enterprise
+
+Consul-Terraform-Sync Enterprise features address organization complexities of collaboration, operations, scale, and governance. Please see the sidebar navigation on the left to view all enterprise features.
```

Comment:
Stole this language from the [Consul Enterprise overview page](https://www.consul.io/docs/enterprise). Open to feedback on language!